docs/data/datasets/datamanagement.md

```diff
@@ -81,6 +81,24 @@ Data sharing is increasingly required by funding agencies and journals as it ben
     - [CSC's videos on Research Data Management](https://video.csc.fi/playlist/details/0_xtsuml9w)  
 
 
+## Budgeting Research Data Management
+
+The processes of data management may incur costs. Researchers should plan to address these costs and the allocation of resources in the early planning phases of the project. The majority of funders accept data management as a legitimate cost that can and should be accounted for in the project budget.
+
+Estimate the resources needed to manage, share and preserve the data. Consider the additional computational facilities and resources that need to be accessed.
+
+Budgeting and costing for your project is often dependent upon institutional resources, services, and policies. Remember to consult your organisation’s data support office.
+
+**Examples of possible data management costs**
```

Comment:
Looks like this renders as a source code element.

https://csc-guide-preview.rahtiapp.fi/origin/siirif-patch-1/data/datasets/datamanagement/#budgeting-research-data-management
![image](https://user-images.githubusercontent.com/50655931/197725144-06e9b87f-ca20-4e47-b22f-655bb01680dc.png)

I'd suggest turning it into an 'admonition':
```suggestion
!!! note "Examples of possible data management costs"
```

Reply:
Thats ok with me.